.github/workflows/main.yml

```diff
@@ -207,7 +207,7 @@ jobs:
         submodules: true
     - uses: ./.github/actions/install-rust
       with:
-        toolchain: nightly-2019-08-15
+        toolchain: nightly-2020-01-06
```

Comment:
Aside: do we still need a dated nightly now that rustup walks back to a nightly with commonly used components that built successfully?

Reply:
It mostly just depends on how cavalier we're feeling. If we follow nightly exactly we'll get breakage from time to time on CI (nightly regressions and such) and while they're typically quickly fixed it may take on the order of a few days to fix nightly.

I think the two strategies are either:

* Pick a nightly and update it deliberately as needed
* Use `nightly` and then when you see breakage pin it to yesterday. When the breakage is fixed go back to `nightly`

The first option is very low maintenance but tends to have nightly lag quite awhile. The latter option can surprise contributors/developers when they submit PRs with unrelated breakage.

I'm fine with either way myself, but the update here was needed to get a build where `#[non_exhaustive]` was stable.

Reply:
I'm fine with dated for now (and especially for this PR), but I agree this very easily gets stale without some automated process to that occasionally bumps it in CI.